scripts/linters/custom_eslint_checks/rules/protractor-practices.spec.js

```diff
@@ -155,5 +161,15 @@ ruleTester.run('protractor-practices', rule, {
         message: 'Please do not use await for "get()"',
       }],
     },
+    {
+      code:
+      `var books = [ 'book1', 'book2', 'book3'];
+      books.forEach( function(book) {
```

Comment:
```suggestion
      `var books = ['book1', 'book2', 'book3'];
      books.forEach(function(book) {
```

Reply:
Can we just make a configuration file for any VScode extension for code formatting to make these spacing and indentation correct? So, we don't have to keep this in mind while writing the code.

Reply:
We've discussed it, but it hasn't been high-priority enough for anyone to decide to work on it yet